streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -792,7 +792,13 @@ private boolean populateClientStatesMap(final Map<UUID, ClientState> clientState
                     log.debug("Fetched end offsets did not contain the changelog {} of task {}", changelog, task);
                     throw new IllegalStateException("Could not get end offset for " + changelog);
                 }
-                taskEndOffsetSums.computeIfPresent(task, (id, curOffsetSum) -> curOffsetSum + offsetResult.offset());
```

Comment:
This is just a side bug that `StreamsPartitionAssignorTest` caught, we need to pin the overflow here too